docs/web-app/trade/more-on-order-types.md

```diff
@@ -0,0 +1,85 @@
+:::
+
+
+## Immediate or Cancel (IOC)
+While it may seem similar to a market order, there is a fundamental difference.
+
+* With an IOC order, you set a specific limit price at which you want the order to be executed.
+* The order will either be immediately filled at that exact price, fully or partially, or canceled entirely. It's about ensuring that your order is executed at your desired price, or not executed at all.
+* The IOC order **does not allow for slippage**, meaning it won't be filled at a different price than what you specified (unlike a market order).
+
+:::info Example
+You place an IOC order to buy 1 WETH at a max price limit of 1,800 USDC.
+Your order could either:
+* Be fully taken immediately, i.e. you get your 1 WETH at the desired price
+* Be partially taken immediately, i.e. you get 0.8 WETH at the desired price, and the rest is "canceled" (there is no resting order asking for the remainder)
+* Be canceled if there is no match on the order book
+:::
+
+## Good 'til time (GTT)
+
+It allows you to set an expiry date for your limit order (ex: active for 3 days, then canceled if not filled).
+A GTT order that was created, but that has not yet been filled will always show as "Filled" in the UI - it is a normal behavior.<br />
+
+Let us explain:
+
+* When placing a GTT order, Mangrove will try to fill it entirely at the desired price.
+
+* Instant order: that is the first execution of the order. If it is a match and the fill is in full, it will be recorded in the UI.
+
+* Resting order: if it can't be matched just yet, a fill at quantity "0" will be recorded in the UI for the instant order, and a resting order will be created.
+    * That allows you to track in the UI that the order was successfully executed, even if no actual quantity was **yet** bought or sold.
+    * It is crucial - without this, there would be no visual trace of what happened to the order you just placed (if not taken). 
+    * Your resting order is then waiting to be filled. There could potentially be multiple fills for that same order. Typically, there will be one fill for the initial order (0 or any other amount taken), and  additional fills up until the resting order is fully taken.
+
+* It ensures that even if there are partial fills or subsequent trades on the resting order, each execution is recorded separately.
+
+:::info Example
+You place an GTT order to buy 1 WETH at a max price limit of 1,800 USDC, with a time limit of 3 days.
+Your order could either:
+* Be fully taken immediately, i.e. you get your 1 WETH at the desired price. A fill of the transaction will be recorded in the UI.
+* Not be taken immediately. A fill at quantity "0" will be recorded in the UI, and your order becomes "resting", waiting to be taken, either fully or partially. Each transaction will log a new fill until the order is fully taken.
+* Be canceled if there is no match on the order book at the end of the set period.

```

Comment:
I took the liberty of rewriting a little bit, you can take what you want from that, because I thought an important possibility was missing which is the "instant partial fill + resting"

```suggestion
* Instant Full Fill: Your order is fully taken right away, meaning you obtain your desired 1 WETH at the specified price. A fill of the transaction will be recorded in the UI.
* Instant Partial Fill and Resting Order: If your order is not entirely filled, a fill with the quantity that has been partially executed will be recorded in the UI. Then, a new order will be created to capture the remaining quantity at the requested price. This order will be resting on the order book, waiting to be fully or partially taken, until the expiry date. Each subsequent transaction that matches your resting order will be logged as a new fill until the order is fully executed or expired.
* No Instant Fill and Resting Order: If there is no matching order available on the order book at the time of placement. In such cases, a "resting" order is posted in the book, waiting to be taken, either fully or partially. Each transaction will log a new fill until the order is fully taken or expired.
* Order Cancellation: If there is no match for your order on the order book by the end of the specified time period (3 days in this case), the order will be automatically canceled.
```